.github/workflows/Linux-pack.yml

```diff
@@ -2,15 +2,15 @@ name: Packaging(Linux)
 
 on:
   push:
-    branches: 
-      - master
+    branches:
+      - master*
```

Comment:
Why do we need wildcards here?

Reply:
I did it for testing, sometimes I need to check installation without merging it master. For example "master_test".